packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -191,7 +218,7 @@
       {
         "type": "apex-replay",
         "label": "Apex Replay Debugger",
-        "program": "./node_modules/@salesforce/salesforcedx-apex-replay-debugger/out/src/adapter/apexReplayDebug.js",
+        "program": "./dist/apexreplaydebug.js",
```

Comment:
oh! this is probably why integration tests were requiring a bundle.  I'll see what I can do about it.